filter-readme-embedded-graphics-core.sed

```diff
@@ -4,3 +4,8 @@
 s#\[(.+)\]\(\.\/(.*)\)#[\1](https://docs.rs/embedded-graphics-core/latest/embedded_graphics_core/\2)#g
 s#\[(.+)\]\: +\.\/(.*)#[\1]: https://docs.rs/embedded-graphics-core/latest/embedded_graphics_core/\2#g
 
+# Delete lines containing reference links like `[Foo]: bar::Foo`
+/\[([^]]+)\]\: ([A-Za-z_]+::)*[A-Za-z_]+$/d
```

Comment:
This means that:

a. We don't have duplicate links in the README source and
b. The broken reference checker actually sees a broken reference if there's a typo between the original link and the reference. This is required as `lychee` happily ignores references to crate paths.

Reply:
I like it, that's cleaner than keeping the unused links in the README.